src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      name="FormSearchBig"
+      class="form-search form-search--big"
+      data-qa="big"
+    >
+      <input
+        class="input__search-bar input__search-bar--big"
```

Comment:
the parent container for this element is `form-search`, so why do you name it `input__search-bar`? maybe better let's `search-bar__input`?